PROCESS.md

```diff
@@ -119,6 +119,6 @@ rebase with master to see if that solves the issue.
 ### Closing down the issue tracker on GitHub
 
 We are currently in the process of closing down the issue tracker on GitHub, to
-prevent duplication with the [GitLab.com issue tracker][https://gitlab.com/gitlab-org/gitlab-ce/issues].
+prevent duplication with the [GitLab.com issue tracker](https://gitlab.com/gitlab-org/gitlab-ce/issues).
```

Comment:
@jvanbaarsen Please escape the braces to make the text to work with copy&paste. Right now those links are rendered has links.